src/qml/pages/MainPage.qml

```diff
@@ -61,13 +61,16 @@ Page {
                     }
 
                     Text {
+                        id: messageBody
                         color: Theme.secondaryTextColor
                         font.pointSize: UiHelper.fixFontSz(12)
-                        text: model.lastMessageBody
                         width: parent.width
+                        text: model.lastMessageBody
                         elide: Text.ElideRight
-                        textFormat: Text.RichText
+//                        textFormat: Text.RichText
+                        maximumLineCount: 1
                     }
+
```

Comment:
There is no need to add extra new line between 2 closing brackets. Seems it left after removing of textClipper().
That's why sometimes it's better to undo commits by git, not manually.

Reply:
Removed